apps/ui/src/views/Bridge/components/BridgeHistory/expandRowContent.tsx

```diff
@@ -0,0 +1,49 @@
+import { BridgeTransactionStatus } from '@force-bridge/commons/lib/types/apiv1';
```

Comment:
Component file name should be PascalCase

Reply:
fixed